vm/src/object/ext.rs

```diff
@@ -1,3 +1,5 @@
+use crossbeam_utils::atomic::AtomicCell;
+
```

Comment:
```suggestion
```
let's not do this. I found everybody has different standard to make order of import sections

Reply:
Oh, that is done by rust-analyzer :)